.gitignore

```diff
@@ -10,14 +10,15 @@
 *.hidden
 *.pkl
 
-# don't ignore important .txt and .csv files
+# don't ignore important .txt, .csv and json files
 !requirements*
 !LICENSE.txt
 !CMakeLists.txt
 !pybamm/CITATIONS.bib
 !pybamm/input/**/*.csv
 !tests/unit/test_parameters/*.csv
 !benchmarks/benchmark_images/*.png
+!pybamm/input/comsol_results/*.json
```

Comment:
The folder is probably already not-ignored by the lines

```gitignore
!input/comsol_results/
```

present below, would you be able to refine this file for redundant rules?

Reply:
Sure, working on cleaning it up!